portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,207 @@
-#content {
+div {
+position:relative;
+}
+
+#nav {
+position: absolute;
```

Comment:
add a 2 space . It makes for much easier readbility. It's great when you come back to this code a while later for readability.